journal-of-systematic-palaeontology.csl

```diff
@@ -171,6 +183,7 @@
       <key macro="author-count" names-min="3" names-use-first="3"/>
       <key macro="author" names-min="3" names-use-first="1"/>
       <key macro="year-date"/>
+     <key macro="month-date"/>
```

Comment:
It might be more straightforward to use a dedicated macro for year/month sorting, e.g.:

```
  <macro name="year-month">
    <date variable="issued">
      <date-part name="year"/>
      <date-part name="month"/>
    </date>
  </macro>
...
       <key macro="year-month"/>
```

(I don't think you need the "no date" bit for sorting)